src/Our.FeatureFlags/App_Plugins/Our.FeatureFlags/package.manifest

```diff
@@ -1,3 +1,4 @@
 {
-  "javascript": [ "~/App_Plugins/Our.FeatureFlags/featureFlags.js" ]
-}
\ No newline at end of file
+  "name": "Our.FeatureFlags",
+  "version": "1.1.0"
```

Comment:
It's for Umbraco's Package Telemetry feature, (since v9.2).
https://our.umbraco.com/Documentation/Extending/Property-Editors/package-manifest#telemetry-elements